openedx/core/djangoapps/credentials/apps.py

```diff
@@ -0,0 +1,34 @@
+"""
+Credentials Configuration
+"""
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+from openedx.core.djangoapps.plugins.constants import ProjectType, SettingsType, PluginURLs, PluginSettings
+
+
+class CredentialsConfig(AppConfig):
+    """
+    Configuration class for credentials Django app
+    """
+    name = 'openedx.core.djangoapps.credentials'
+    verbose_name = _("Credentials")
+
+    plugin_app = {
+        PluginURLs.CONFIG: {
+            ProjectType.LMS: {
+                PluginURLs.NAMESPACE: u'credentials',
+                PluginURLs.REGEX: u'api/credentials/',
+                PluginURLs.RELATIVE_PATH: u'urls',
```

Comment:
if the urls are registered here now we can remove the references from the giant urls.py file